### ISSUE TITLE
Tracking research studies leveraging SCUBA

### ISSUE THREAD

**Issue by patwater** [California-Data-Collaborative/metadata_catalogue#1]
On the statewide data action team meeting today there was a great suggestion to track the studies that are occurring using data from SCUBA. One simple thought would be to track those using GitHub issues, creating a unique issue for each outstanding project to track general information such as A) the research question B) initial timeline C) milestones etc. and D) link to the final research product when available. 

CC @christophertull @datwater

**Comment by christophertull** [comment]
ON a very related note, I have been thinking about quantifying the long term research impact lately. Would be great to be able to say 2 years down the line that CaDC member data and funding had supported:

* X Journal articles
* Y conference papers
* Z dissertations/master theses

Obviously this is not a true measure of "impact" but at least it quantifies one aspect of CaDC outputs.

Less critical but could also be helpful to have something for researchers to cite in their work. This could just be a standard [dataset citation](http://guides.lib.umich.edu/c.php?g=439304&p=2993299), or we could polish the Metadata doc a little and put it up on ArXiv (not sure how we would handle updated versions of the doc) or a new short paper that just serves as a high-level, time-invariant description of SCUBA. Primary idea here would be to have a "citation bucket" that would track the impact for us.

**Comment by patwater** [comment]
What are your thoughts by the way on having an Airbnb knowledge repo to track results?

**Comment by christophertull** [comment]
I think the knowledge repo (KR) is a great idea generally but I think it might be overkill in this case? Afaik the KR requires that posts originate as Rmarkdown or Ipynb files, which is a great assumption for data science folks, and if you have a whole team already working in this format then it makes sense to take that work and "blogify" it.

However, in our case with external researchers, I think it might just add overhead.

**Comment by patwater** [comment]
Yeah I concur. KISS is the name of the game. I think a knowledge repo (death to acronyms, the mind killers) makes the most sense for internal ad hoc analyses particularly under a <many> data collaboratives scenario and providing inter-collab QA/QC

**Comment by christophertull** [comment]
> KISS ... (death to acronyms, the mind killers) ... QA/QC

;)

**Comment by patwater** [comment]
Quick note to flag that we have executed the data sharing with RFF and UCSC.

**Comment by patwater** [comment]
Current studies as of July 2, 2018:

- John / Arisha / Holly Mission Viejo
- RFF Econ
- UCSC Econ
- Stanford marketing
- USC Integrated Demand / Supply

**Comment by patwater** [comment]
Updates December 2018

- Mission Viejo -- currently developing a training curriculum for google streetview to expand that dataset with volunteers from Hack for LA / MapTimeLA 
- UCSC -- preliminary results for a subset of districts and working on follow up questions / expanded dataset
- RFF -- dealing with data issues
- Stanford (in hibernation)
- USC -- planning to include in January 29th steering committee meeting

*Future work discussed*
- Evaluate the impact of price and non-price interventions across socioeconomic subgroups and different profiles of previous conservation
- Evaluate the impact of the conservation programs targeted to new customers moving in (welcome packet and audit).

CC @christophertull @dmarulli @vr00n
